src/_tests/fixtures/44105/_owners.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Prefixing owners filename with an underscore because it’s an input, like `_response.json`